rzup/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "rzup"
+version = "0.2.1"
+edition = { workspace = true }
+license = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+
+[dependencies]
+anyhow = "1.0.86"
+cfg-if = "1.0.0"
+clap = { version = "4.5.7", features = ["derive"] }
+dirs = "5.0.1"
+flate2 = "1.0.30"
+fs2 = "0.4.3"
+fs_extra = "1.3.0"
+lazy_static = "1.5.0"
+regex = "1.10.5"
+reqwest = { version = "0.12.5", features = ["json", "rustls-tls"] }
+serde = { version = "1.0.203", features = ["derive"] }
+serde_json = "1.0.117"
+tar = "0.4.41"
+tempfile = "3.10.1"
+termcolor = "1.4.1"
+thiserror = "1.0.61"
+tokio = { version = "1.38.0", features = ["full"] }
```

Comment:
Do we actually need `full`? Seems like a heavy dep.

Reply:
Good call out - we can reduce this to just `macros` and `rt-multi-thread`.